### PR TITLE
Prep for Creation of 8.3 Branch

### PR DESCRIPTION
**Pull request by terrancedejesus**
## Resolves
elastic/ia-trade-team/issues/27

## Summary
- [x] update packages.yml
- [x] update workflows
- [x] lock api_schemas
- [x] add migrate function
- [x] update stack-schema-map

_merged after 8.2 branch creation_

### CODE REVIEW

File: etc/packages.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ package:
   registry_data:
     categories: ["security"]
     conditions:
-      kibana.version: "^8.2.0"
+      kibana.version: "^8.3.0"
```
Comment (brokensound77) [inline etc/packages.yml:29]: also bump version below to `1.3.0-dev.0`

**Review by brokensound77 — APPROVED** [review]
LGTM, thanks

**Review by w0rk3r — APPROVED** [review]

File: etc/stack-schema-map.yaml [hunk 2]
```diff
@@ -34,3 +34,7 @@
 "8.2.0":
   beats: "main"  # TODO: update this once beats releases
   ecs: "1.12.1"
+
```
Comment (Mikaayenson) [inline etc/stack-schema-map.yaml:37]: We should remove the other `update this once beats releases` comments and make sure the versions are correct for older releases.

File: .github/workflows/backport.yml [hunk 3]
```diff
@@ -56,7 +56,7 @@ jobs:
       max-parallel: 1
       matrix:
         # 7.17 was intentionally skipped because it was added late and was bug fix only
-        target_branch: [7.13, 7.14, 7.15, 7.16, '8.0', 8.1]
+        target_branch: [7.13, 7.14, 7.15, 7.16, '8.0', 8.1, 8.2]
```
Comment (Mikaayenson) [inline .github/workflows/backport.yml:59]: Shouldn't we drop support for some of these?

File: etc/stack-schema-map.yaml (hunk 2 above)
Comment (brokensound77) [reply etc/stack-schema-map.yaml:37]: we can refresh the ones that are available and update in a separate pr 👍

File: .github/workflows/backport.yml (hunk 3 above)
Comment (brokensound77) [reply .github/workflows/backport.yml:59]: like we discussed, we should def consider it, but we would should do some additional review to make sure everything is aligned

**Pull request merged**